app/streams/publishing_api/message_adapter.rb

```diff
@@ -63,6 +63,12 @@ def attributes
         public_updated_at: parse_time('public_updated_at'),
         schema_name: message.payload.fetch('schema_name'),
         raw_json: message.payload,
+        phase: message.payload.fetch('phase', nil),
+        publishing_app: message.payload.fetch('publishing_app', nil),
+        rendering_app: message.payload.fetch('rendering_app', nil),
+        analytics_identifier: message.payload.fetch('analytics_identifier', nil),
+        update_type: message.payload.fetch('update_type', nil),
+        links: message.payload.fetch('links', nil),
```

Comment:
I would rather storing the `expanded_links` instead of the `links`. This way we also store the exact representation of those entities when the item is updated.

If you rename the attribute to `expanded_links` you would also be able [to remove the method ](https://github.com/alphagov/content-performance-manager/blob/13642c1e0dc18d633c84ba9aae1f706cfc449842/app/models/dimensions/item.rb#L30) that is currently wrapping the `raw_json`. 

On the same go you would also be able to remove `raw_json`?